File: 프로그래머스/1/12919. 서울에서 김서방 찾기/서울에서 김서방 찾기.cs
using System;

public class Solution 
{
    public string solution(string[] seoul) 
    {
        string ans = "";
        for(int i=0; i<seoul.Length; i++)
        {
            if(seoul[i] == "Kim")
            {
                ans = $"김서방은 {i}에 있다";
                break;
            }
        }
        return ans;
    }
}

File: 프로그래머스/1/12919. 서울에서 김서방 찾기/서울에서 김서방 찾기.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

string solution(vector<string> seoul) 
{
    auto it = find(seoul.begin(), seoul.end(), "Kim");
    int pos = distance(seoul.begin(), it);
    string p = to_string(pos);
    string ans = "김서방은 " + p + "에 있다";
    return ans;
}